src/order-book/api.ts

```diff
@@ -46,47 +39,31 @@ export const ORDER_BOOK_STAGING_CONFIG: ApiBaseUrls = {
   [SupportedChainId.GOERLI]: 'https://barn.api.cow.fi/goerli',
 }
 
-class FetchHttpRequest extends BaseHttpRequest {
-  constructor(config: OpenAPIConfig) {
-    super(config)
-  }
-
-  /**
-   * Request method
-   * @param options The request options from the service
-   * @returns CancelablePromise<T>
-   * @throws ApiError
-   */
-  public override request<T>(options: ApiRequestOptions): CancelablePromise<T> {
-    return __request(this.config, {
-      ...options,
-      headers: {
-        ...options.headers,
-        'Content-Type': 'application/json',
-      },
-    })
-  }
-}
-
 export class OrderBookApi {
   public context: ApiContext
-  private servicePerNetwork: Record<string, OrderBookClient | null> = {}
 
-  constructor(context: PartialApiContext = {}) {
+  private rateLimiter: RateLimiter
+
+  constructor(
+    context: PartialApiContext = {},
+    limiterOpts: RateLimiterOpts = DEFAULT_LIMITER_OPTIONS,
+    public readonly backoffOpts: BackoffOptions = DEFAULT_BACKOFF_OPTIONS
+  ) {
```

Comment:
Why are these options not part of the `context` option obj?

I personally prefer to have one single obj parameter, so if you want to omit some parameter and only provide the last, you don't need to pass `undefined` for them

Reply:
I though about it :) Thanks! Fixed